docs/reference/remap/functions/decode_base64.cue

```diff
@@ -13,19 +13,51 @@ remap: functions: decode_base64: {
 			required:    true
 			type: ["string"]
 		},
+		{
+			name:        "padding"
+			description: "Whether the Base64 output is [padded](\(urls.base64_padding))."
```

Comment:
Is this needed? I don't think decoding needs to know a priori whether or not there is padding.

https://docs.rs/base64/0.13.0/base64/fn.decode_config.html shows an example of just decoding with the charset.

Reply:
You're right. I experimented with this a bunch and setting padding vs. no padding has no effect.